src/imagery/i.sentinel/i.sentinel3.import/i.sentinel3.import.html

```diff
@@ -0,0 +1,131 @@
+<h2>DESCRIPTION</h2>
+
+The <em>i.sentinel3.import</em> module allows importing Copernicus Sentinel-3 products
+downloaded by the <em><a href="i.sentinel.download.html">i.sentinel.download</a></em>
+module.
+
+<p>
+By default <em>i.sentinel3.import</em> imports all Sentinel-3 scene files found
+in the <b>input</b> directory. The number of scene files can be optionally
+reduced by the <b>pattern</b> option or filtered by modification time using the 
+<b>modified_after</b> and/or <b>modified_before</b> option. In the <b>pattern</b> 
+option, a regular expression for filtering the file names should be given, e.g. 
+"0179_076_100_0900_LN2" for importing only specific tiles.
+<p>
+The <em>Sentinel-3</em> format is currently not supported by GDAL and consists 
+of several netCDF4 files. Therefore, a GRASS GIS specific import routine is implemented. 
+It imports the data pixel-wise after transforming them from WGS84 into the projection 
+of the current LOCATION. <em>r.in.xyz<em> is then used to read the coordinates into 
+a GRASS GIS raster map. Thus, the the user has to set the computational region extent and 
+resolution appropriately, to import the data of interest in the correct resolution.
+
+<p>
+By default, ancillary data and quality flag data is imported as well - as artificial 
+bands. Import of those additional data layers can be deactivated using the <b>ancillary_bands</b> 
+and <b>quality_bands</b> option.
+
+<p>
+For each imported band both scene and band specific metadata  are written into the map history
+(<em><a href="https://grass.osgeo.org/grass-stable/manuals/r.support.html">r.support</a></em>). 
+In addition, the scene name is stored as <em>source1</em> and the imported or linked file name as
+<em>source2</em>. Also, sensing time is written into the timestamp of the
+map. After import, the metadata can be retrieved with <em>r.info -e</em>
+as shown below.
+
+
+<h2>NOTES</h2>
+
+<p>
+Currently, only import of SL_2_LST__ products is supported.
+<p>
+By <b>register_file</b> option <em>i.sentinel3.import</em> allows
+creating a file which can be used to register imported imagery data
+into space-time raster dateset (STRDS) with 
+<em><a href="https://grass.osgeo.org/grass-stable/manuals/t.register.html">t.register</a></em>. 
+See example below.
+
+<p>
+<h3>Metadata storage</h3>
+
+By using the <b>-j</b> flag the band metadata are additionally stored
+in JSON format (in the current mapset under <tt>cell_misc</tt>).
+
+
+<h2>EXAMPLES</h2>
+
+<h3>List Sentinel bands</h3>
+
+At first, print list of raster files to be imported by <b>-p</b>. For
+each file also product content is printed:
+
+<div class="code"><pre>
+i.sentinel3.import -p input="data" nprocs=4 modified_before="2021-09-09" product=LST basename=S3_LST
+
+...
```

Comment:
Why "data"? Are the quotes necessary there? Also, maybe put some of the resulting printed info as example and not only the `...` ;)